packages/rke2-calico/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://github.com/projectcalico/calico/releases/download/v3.24.1/tigera-operator-v3.24.1.tgz
-packageVersion: 03
```

Comment:
If there is a new url, the packageVersion moves back to 01

Reply:
done, thanks